.pre-commit-config.yaml

```diff
@@ -36,7 +35,7 @@ repos:
     rev: 5.12.0
     hooks:
       - id: isort
-        exclude: src
+        exclude: src/polus/plugins/_plugins/models/PolusComputeSchema.py|src/polus/plugins/_plugins/models/WIPPPluginSchema.py
```

Comment:
I think we can just put these on separate lines, right?

Same with the above configurations.

Reply:
Solved with regex in one shorter linee